docs/deployment/kubernetes.md

```diff
@@ -16,41 +16,49 @@ To deploy Quickwit on Kubernetes, you will need:
 - kubectl, compatible with your cluster (+/- 1 minor release from your cluster) (`kubectl version`)
 - Helm v3 (`helm version`)
 - A Kubernetes cluster
+- An instance of PostgreSQL server
```

Comment:
You can have a quickwit cluster without postgresql server, a file backed metastore on S3 works. The current helm chart needs to be clearer on that, there is an open PR to ease the setup of the S3 metastore https://github.com/quickwit-oss/helm-charts/pull/43

```suggestion
- An instance of PostgreSQL server or an S3-backed metastore
```

Reply:
For now, you have to set QW_METASTORE_URI env variable to make it work with an S3 metastore.

Reply:
Hmm. That will simplify things quite a bit at the expense of performance. Let me play with it. Maybe we can add file store as the main path and move postgresql to the end as a scalable but more complicated alternative.